nova6/nova6dl.py

```diff
@@ -32,16 +32,15 @@
 from __future__ import print_function, unicode_literals
 
 import argparse
-import sys
 from .helpers import download_file
 from .nova6 import get_engines
 
 def parse_args(args=None):
     parser = argparse.ArgumentParser(description="Downloads a torrent from a website and stores it to a temporary file")
     parser.add_argument('--engines-dir', '-d', default=None, dest="engines_dirs", action='append',
                         help='Specify custom directory for engine plugins. Default is the engines directory inside of the current script directory. Can be specified multiple times')
-    parser.add_argument('engine_url', nargs=1, default=None, help='The search engine URL returned from nova2')
-    parser.add_argument('download_parameter', nargs=1, default=None, help='The download parameter')
+    parser.add_argument('engine_url', help='The search engine URL returned from nova2')
+    parser.add_argument('download_parameter', help='The download parameter')
```

Comment:
I'll take for granted that this is the default, I usually tend to be specific in case API changes (maybe they decide to change it in a newer Python version).

Reply:
The thing is that you don't really need default arguments for required parameters. Otherwise they should be optional